cypress/integration/seededFlows/adminFlows/config/authenticationSection.spec.js

```diff
@@ -31,11 +31,6 @@ describe('Authentication Section', () => {
             .should('not.be.checked')
             .check();
 
-          cy.get('@authSectionForm')
```

Comment:
I wonder if we should add some new tests for the error paths? We can use Cypress to fake error responses from the server -  e.g. we use intercepts to check some error paths in comment creation

https://github.com/forem/forem/blob/b58fdd013b6e362fe5e120decb5ab9f0cec00375/cypress/integration/seededFlows/articleFlows/commentOnArticle.spec.js#L468-L470

what do you think? Maybe not needed for _every_ input 😅

Reply:
I think it's a good idea. I will add some.